opm/output/eclipse/Summary.cpp

```diff
@@ -748,6 +748,9 @@ inline quantity glir( const fn_args& args ) {
 
         const auto& production = well->productionControls(args.st);
         if (! has_vfp_table(sched_state, production.vfp_table_number)) {
+            const double eff_fac = efac(args.eff_factors, well->name());
+            alq_rate += eff_fac * xwPos->second.rates.get(rt::alq, production.alq_value);
+            return {alq_rate, measure::identity };
             continue;
```

Comment:
The `return` statement here means that the `continue` statement is redundant.

Reply:
Ah.. should be continue, see below.